.solhint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "solhint:default"
```

Comment:
`"solhint:default"` checks almost nothing.  This is just to get us started.  We probably want to get to `"solhint:recommended"` with some exceptions, but it will take some work to get there:
```
$ make lint-solidity | tail -2
make: *** [Makefile:26: lint-solidity] Error 1
✖ 52 problems (10 errors, 42 warnings)
```